lib/atom-scalariform.js

```diff
@@ -5,6 +5,19 @@ var packagePath = function() {
   return atom.packages.getLoadedPackage('atom-scalariform').path;
 }
 
+var projectPath = function(fileToFormatPath) {
+  var ref = atom.project.getDirectories();
+  var projectPath = '';
+  for (var i = 0, len = ref.length; i < len; i++) {
+    var directory = ref[i];
+    if (fileToFormatPath.indexOf(directory.path) > -1) {
```

Comment:
Not really, `directory.path` will only include the absolute path to the project folder, but the `scala` file to format might be in a subdirectory there:
* Project: /home/user/myProject
* File: /home/user/myProject/folder1/subfolder1/Test.scala

The only way that you could get a false positive would be:
* Actual Project: /home/user/myProject
* Other Project: /home/user/myOtherProject
* File: /home/user/myProject/home/user/myOtherProject/Test.scala

Am I missing any other corner case?

Maybe I just need to force `indexOf` to be `0`:
```js
fileToFormatPath.indexOf(directory.path) === 0
```

What do you think?